cmip6_downscaling/methods/common/tasks.py

```diff
@@ -144,16 +144,26 @@ def get_experiment(run_parameters: RunParameters, time_subset: str) -> UPath:
 
 
 @task(log_stdout=True)
-def rechunk(path: UPath, chunking_pattern: Union[str, UPath] = None, max_mem: str = "2GB") -> UPath:
+def rechunk(
+    path: UPath,
+    chunking_pattern: str = None,
+    chunking_template_file: UPath = None,
+    max_mem: str = "2GB",
+) -> UPath:
     """Use `rechunker` package to adjust chunks of dataset to a form
     conducive for your processing.
 
     Parameters
     ----------
     path : UPath
         path to zarr store
-    chunking_pattern : str or UPath
-        The pattern of chunking you want to use.
+    chunking_pattern : str
+        The pattern of chunking you want to use. If used together with `chunking_template_file` it will override the template
+        to ensure that the final dataset truly follows that `full_space` or `full_time` spec. This matters when you are passing
+        a template that is either a shorter time length or a template that is a coarser grid (and thus a shorter lat/lon chunksize)
+    chunking_template_file : UPath
```

Comment:
```suggestion
    pattern : str
        The pattern of chunking you want to use. If used together with `template` it will override the template
        to ensure that the final dataset truly follows that `full_space` or `full_time` spec. This matters when you are passing
        a template that is either a shorter time length or a template that is a coarser grid (and thus a shorter lat/lon chunksize)
    template : UPath
```